planetscale/branches.go

```diff
@@ -271,6 +300,54 @@ func (d *databaseBranchesService) VSchema(ctx context.Context, vSchemaReq *Branc
 	return vSchema, nil
 }
 
+func (d *databaseBranchesService) UpdateVSchema(ctx context.Context, updateVSchemaReq *UpdateBranchVschemaRequest) (*VSchemaDiff, error) {
+	path := fmt.Sprintf("%s/vschema", databaseBranchAPIPath(updateVSchemaReq.Organization, updateVSchemaReq.Database, updateVSchemaReq.Branch))
+
+	req, err := d.client.newRequest(http.MethodPatch, path, updateVSchemaReq)
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating http request")
+	}
+
+	vSchema := &VSchemaDiff{}
```

Comment:
@nickvanw I think this type might be named wrong? Is this really a diff?

Reply:
I did not want to change it in this PR, but it is not a diff - I believe the type is named improperly.